src/pybind/mgr/dashboard_v2/frontend/src/app/app-routing.module.ts

```diff
@@ -1,17 +1,23 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { DashboardComponent } from './ceph/dashboard/dashboard/dashboard.component';
 import { AuthGuardService } from './shared/services/auth-guard.service';
 import { LoginComponent } from './core/auth/login/login.component';
 import { HostsComponent } from './ceph/cluster/hosts/hosts.component';
 
 const routes: Routes = [
-  { path: '', redirectTo: 'hosts', pathMatch: 'full' },
+  { path: '', redirectTo: 'dashboard', pathMatch: 'full' },
+  {
+    path: 'dashboard',
+    canActivate: [AuthGuardService],
+    component: DashboardComponent
```

Comment:
Can you switch line 12 and 13 to use the same order that we use on `login` and `hosts` (1 path, 2 component, 3 canActivate)?

Reply:
done